plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ContractDescriptionConversionVisitor.kt

```diff
@@ -17,7 +18,10 @@ import org.jetbrains.kotlin.formver.embeddings.SourceRole
 import org.jetbrains.kotlin.formver.embeddings.callables.NamedFunctionSignature
 import org.jetbrains.kotlin.formver.embeddings.expression.*
 
-class ContractVisitorContext(val returnVariable: VariableEmbedding)
+data class ContractVisitorContext(
+    val returnVariable: VariableEmbedding,
+    val targetParameterFunction: FirValueParameterSymbol? = null
```

Comment:
I'm not sure this name communicates what you may want it to; "parameter function" makes it sound like it's a function of the parameter.  Why not just `targetParameter` and a comment that makes it clear this is non-null for callsInPlace effects?